src/functions/index.ts

```diff
@@ -24,4 +24,5 @@ export { default as deleteSoftwareHandler } from "./software-registry/delete-sof
 export { default as listUsersHandler } from "./keycloak/list-users";
 export { default as findUserHandler } from "./keycloak/find-user";
 export { default as createQuestionnaireHandler } from "./questionnaire/create-questionnaire";
-export { default as findQuestionnaireHandler } from "./questionnaire/find-questionnaire";
\ No newline at end of file
+export { default as findQuestionnaireHandler } from "./questionnaire/find-questionnaire";
+export { default as deleteQuestionnaireHandler } from "./questionnaire/delete-questionnaire";
```

Comment:
Is this PR also for list questionnaire? In which case it should be exported from here. Also probably should have been included in the branch name- just something to consider for the future